hangman/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿//homework-hangman or guessing words
+
+//pick a word randomly from a file, and let users to guess and users get the hint
+
+
+using System;
+using System.IO;
+
+namespace hangman
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string[] lines =
+            System.IO.File.ReadAllLines(@"C:\Users\jungi\Desktop\words.txt");//read everything
+                                                                             //randomly select a line
+            Random random = new Random();
+            int r = random.Next(lines.Length);
+            string secretWord = lines[r];//gets the string word at the random r array
+            Console.WriteLine("The secret word is " + secretWord);
+            Console.WriteLine("Guess a word");
+            string s = Console.ReadLine();
+            int intGuess = Array.IndexOf(lines, s);//get the index of the object in the array lines
```

Comment:
This works as long as the word i guess is also in the file. But what if the word i am guessing is not in the file?